tests/test_suppression.py

```diff
@@ -132,3 +133,26 @@ def test_single_row():
 
     mask = zen._do_suppression(df, low=1, high=5)
     assert (mask.values == expected_array).all()
+
+def test_nan_in_mean_sd_table():
+    total_length = 20
+    cuisine = ['Chinese', 'Korean', 'Italian']
+    city = ['Boston', 'Providence']
+    who = ['Paul', 'Ed', 'Kevin']
+
+    # Set seed to make this example reproducible
+    d = {"city": random.choices(city, k=total_length),
```

Comment:
Is meant to deliberately create empty groupings or not? As it stands, I can't tell what the input to the test will be, which is somewhat critical when dealing with thresholds. :-)